onnxruntime/core/graph/abi_graph_types.h

```diff
@@ -251,6 +251,16 @@ struct OrtNode {
   /// <returns>A status indicating success or an error.</returns>
   virtual onnxruntime::Status GetAttributes(gsl::span<const OrtOpAttr*> attrs) const = 0;
 
+  /// <summary>
+  /// Gets the node's 'TENSOR' attribute as an OrtValue.
+  /// </summary>
+  /// <param name="attr">Node's 'TENSOR' attribute.</param>
+  /// <param name="value">Output parameter set to the 'TENSOR' attribute value or nullptr
+  ///                     if it's not a 'TENSOR' attribute.</param>
+  /// <returns>A status indicating success or an error.</returns>
+  virtual onnxruntime::Status GetTensorAttributeAsOrtValue(const OrtOpAttr* attr,
+                                                           OrtValue** value) const = 0;
```

Comment:
update "value" comment - `*value` will no longer get set to nullptr.

also, maybe consider `OrtValue*& value` so we don't have to worry about `value` being nullptr within this function.